### ISSUE TITLE
Improve 'system config file doesn't exist' log message

### ISSUE THREAD

**Issue by ButKor** [thin-edge/thin-edge.io#2797]
**Is your feature improvement request related to a problem? Please describe.**
When doing `tedge connect c8y`, first-time tedge users are regularly stumbling over log line "The system config file '/etc/tedge/system.toml' doesn't exist. Use '/bin/systemctl' as a service manager" and ask what's the problem and how to fix it. So this log is interpreted as warning/error while it's actually just an info and nothing to worry about. 

```sh
$ sudo tedge reconnect c8y
The system config file '/etc/tedge/system.toml' doesn't exist. Use '/bin/systemctl' as a service manager.
```

**Describe the solution you'd like**
Mentioned log line could be rephrased to make clear: this is not an error to worry about and just an information.

**Describe alternatives you've considered**
Maybe it makes sense to print a log severity to the messages on console?

**Comment by didier-wenzek** [comment]
I propose to replace this message by:

```
Using '/bin/systemctl' as no 'init' system has been configured in '/etc/tedge/system.toml'
```

**Comment by reubenmiller** [comment]
This change was included as part of 1.1.0.

The following is the new message displayed to the user.

```sh
The system config file '/etc/tedge/system.toml' doesn't exist. Using '/bin/systemctl' as the default service manager
```
